File: app/src/main/java/com/thinkzi/weather/ui/view/activity/RouteActivity.java
package com.thinkzi.weather.ui.view.activity;

import android.os.Bundle;

import androidx.annotation.Nullable;

/**
 * provide main first Activity
 * */
public class RouteActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.getApplicationComponent().inject(this);
        _navigator.navigateToWeatherInCityUIModelForFiveDaysActivity(this);
        finish();
    }
}

File: app/src/main/java/com/thinkzi/weather/ui/view/activity/RouteActivity.kt
package com.thinkzi.weather.ui.view.activity

import android.os.Bundle

/**
 * provide main first Activity
 */
class RouteActivity : BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        this.applicationComponent!!.inject(this)
        _navigator.navigateToWeatherInCityUIModelForFiveDaysActivity(this)
        finish()
    }
}
